sdk/core/core-http/src/nodeFetchHttpClient.ts

```diff
@@ -119,7 +120,7 @@ export class NodeFetchHttpClient extends FetchHttpClient {
   async processRequest(operationResponse: HttpOperationResponse): Promise<void> {
     if (this.cookieJar) {
       const setCookieHeader = operationResponse.headers.get("Set-Cookie");
-      if (setCookieHeader != undefined) {
+      if (setCookieHeader !== undefined && setCookieHeader !== null) {
```

Comment:
`headers.get()` never returns null, right? It's either the string value or undefined.

Reply:
Agree. I fixed this too.  Will merge after CI is green so we can have nightly live tests coverage for this change.